conf/modules/telemetry_transparent_secure.xml

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="telemetry_transparent_secure" dir="datalink" task="datalink">
+  <doc>
+    <description>
+      Telemetry using secure PPRZ protocol over UART
+
+      Similar to telemetry_transparent, except it adds encryption
+    </description>
+    <configure name="MODEM_PORT" value="UARTx" description="UART where the modem is connected to (UART1, UART2, etc)"/>
+    <configure name="MODEM_BAUD" value="B57600" description="UART baud rate"/>
+  </doc>
+  <autoload name="telemetry" type="nps"/>
+  <autoload name="telemetry" type="sim"/>
+  <header>
+    <file name="spprz_dl.h"/>
+  </header>
+  <init fun="spprz_dl_init()"/>
+  <event fun="spprz_dl_event()"/>
+  <makefile target="!fbw|sim|nps|hitl">
+    <raw>
+      # to not use 128 bit arithmetic
+      ap.CFLAGS += -DKRML_NOUINT128
+    </raw>
+    <configure name="MODEM_PORT" case="upper|lower"/>
+    <define name="USE_$(MODEM_PORT_UPPER)"/>
+    <define name="$(MODEM_PORT_UPPER)_BAUD" value="$(MODEM_BAUD)"/>
+    <define name="DOWNLINK"/>
+    <define name="PERIODIC_TELEMETRY"/>
+    <define name="DOWNLINK_DEVICE" value="$(MODEM_PORT_LOWER)"/>
+    <define name="PPRZ_UART" value="$(MODEM_PORT_LOWER)"/>
+    <define name="DOWNLINK_TRANSPORT" value="spprz_tp"/>
+    <define name="DATALINK" value="PPRZ"/>
+    <file name="spprz_dl.c"/>
+    <file name="downlink.c" dir="subsystems/datalink"/>
+    <file name="datalink.c" dir="subsystems/datalink"/>
+    <file name="telemetry.c" dir="subsystems/datalink"/>
+    <file name="secure_pprz_transport.c" dir="pprzlink/src"/>
+
+    <!-- HACL-C files -->
+    <!-- TODO: convert into a paparazzi module? -->
```

Comment:
this has to be included here, so perhaps having a Paparazzi module would be easier...(it is unlikely that anyone will use secure pprzlink as standalone anyway)